crates/aptos/src/common/utils.rs

```diff
@@ -50,19 +51,28 @@ pub fn prompt_yes(prompt: &str) -> bool {
 }
 
 /// Convert any successful response to Success
-pub async fn to_common_success_result<T>(command: &str, result: CliTypedResult<T>) -> CliResult {
-    to_common_result(command, result.map(|_| "Success")).await
+pub async fn to_common_success_result<T>(
+    command: &str,
+    start_time: SystemTime,
+    result: CliTypedResult<T>,
+) -> CliResult {
+    to_common_result(command, start_time, result.map(|_| "Success")).await
 }
 
 /// For pretty printing outputs in JSON
-pub async fn to_common_result<T: Serialize>(command: &str, result: CliTypedResult<T>) -> CliResult {
+pub async fn to_common_result<T: Serialize>(
+    command: &str,
+    start_time: SystemTime,
+    result: CliTypedResult<T>,
+) -> CliResult {
+    let latency = SystemTime::now().duration_since(start_time).ok();
     let is_err = result.is_err();
     let error = if let Err(ref e) = result {
         e.to_str()
     } else {
         "None"
     };
-    let metrics = collect_metrics(command, !is_err, error);
+    let metrics = collect_metrics(command, !is_err, latency, error);
```

Comment:
are we collecting metrics from users installing the Aptos Cli? or just collect from local Cli. If this is the former, seems not very useful as they may have it installed on different machines?

Reply:
It's from all users, I know there will be a lot of variance, but it would nice to see what the customer experience is in terms of time.

Each of the APIs have different characteristics